README.md

```diff
@@ -91,6 +91,28 @@ jsonLogic(rules, data)
 # True
 ```
 
+It is also possible to specify default values for the vars:
+
+```python
+rules = { "var": ["a", 1] }
+
+data = { "b": 2 }
+
+jsonLogic(rules, data)
+# 1
+```
+
+If the value is present but empty, the default value will be used:
+
+```python
+rules = { "var": ["a", 1] }
+
+data = { "a": None, "b": 2 }
+
+jsonLogic(rules, data)
+# 1
+```
```

Comment:
I'd point out that this is slightly different behaviour than in JS, where the default seems to only be used for `undefined` values:

```node
> logic = {"var": ["a", 3]}
> data = {"a": undefined}
> jsonLogic.apply(logic, data);
3
> data = {"a": null}
> jsonLogic.apply(logic, data);
null
```